securemail/securemail.php

```diff
@@ -141,7 +167,12 @@ function securemail_emailer_send_prepare(&$a, &$b) {
         'filename' => 'encrypted.gpg'
     ));
     $encrypted = OpenPGP_Crypt_Symmetric::encrypt($key, new OpenPGP_Message(array($data)));
-    $armored_encrypted = wordwrap(OpenPGP::enarmor($encrypted->to_bytes(), "PGP MESSAGE"), 64, "\n", true);
+    $armored_encrypted = wordwrap(
+        OpenPGP::enarmor($encrypted->to_bytes(), 'PGP MESSAGE'),
+        64,
+        '\n',
```

Comment:
I believe the intended behavior was to add an actual end of line character, not the literal `\n` characters. Usage of `PHP_EOL` is suggested.